src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+for (let i = 0; i < table.rows.length; i++) {
```

Comment:
Ensure that the table has at least five cells in each row before attempting to insert a new cell before the fifth cell. Otherwise, this may cause an error if a row has fewer than five cells.